### PR TITLE
Add netbase dependency to Ubuntu docker files

### PR DESCRIPTION
**Pull request by Kami**
This pull request adds `netbase` package dependency which is required by st2 and it's missing in some base container images.

### CODE REVIEW

File: packagingenv/trusty/Dockerfile [hunk 1]
```diff
@@ -38,6 +38,9 @@ RUN echo -e "#!/bin/sh\nexit 101\n" > /usr/sbin/policy-rc.d && \
 RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
     apt-get -y install build-essential python-dev python
 
+# install netbase package (includes /etc/protocols and other files we rely on)
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
```
Comment (armab) [inline packagingenv/trusty/Dockerfile]: Please remove another duplicated `apt-get update`. We already have a lot of them.

File: stackstorm/Dockerfile [hunk 2]
```diff
@@ -8,7 +8,7 @@ LABEL com.stackstorm.version="${ST2_VERSION}"
 
 ADD ./sources.list /etc/apt/ 
 
-RUN apt-get -y update && apt-get -y install adduser procps python2.7 libffi5 libyaml-0-2 git libpcre3 libpython2.7 libxml2 python-dev libssl-dev libffi-dev sudo
+RUN apt-get -y update && apt-get -y install adduser procps python2.7 libffi5 libyaml-0-2 git libpcre3 libpython2.7 libxml2 python-dev libssl-dev libffi-dev sudo netbase
```
Comment (armab) [inline stackstorm/Dockerfile]: This change is not needed.
Please remove it to avoid  changelog/confusion.

Comment (Kami) [reply stackstorm/Dockerfile]: Thanks for the heads up - will remove it.

File: packagingenv/trusty/Dockerfile (hunk 1 above)
Comment (Kami) [reply packagingenv/trusty/Dockerfile]: Yeah, wasn't sure about it - I saw it on every line so I just followed the existing approach.

File: packagingbuild/centos7/Dockerfile [hunk 3]
```diff
@@ -11,7 +11,8 @@ RUN yum -y install \
     git \
     mercurial \
     openssh \
-    subversion
+    subversion \
+    setup
```
Comment (armab) [inline packagingbuild/centos7/Dockerfile]: Don't think we need this rpm dependency during the st2 Package build (`packagingbuild`).
However we'll need it during the st2 Package tests (`packagingtest`).

**Pull request merged**